Gemfile

```diff
@@ -3,8 +3,8 @@
 
 source "https://rubygems.org"
 gem "logstash-core", "3.0.0.dev", :path => "./logstash-core"
-# gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
-gem "logstash-core-event-java", "3.0.0.dev", :path => "./logstash-core-event-java"
+gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
```

Comment:
well, this is what we have in 2.x, when we backport in 2.3 we have to adjust anyway so, I don't see the need to change that here at this point.